core/src/actions/config.ts

```diff
@@ -69,3 +71,24 @@ export class ConfigGenerate extends AuthenticatedAction {
     ]);
   }
 }
+
+export class ConfigUserCreate extends OptionallyAuthenticatedAction {
+  constructor() {
+    super();
+    this.name = "config:user:create";
+    this.description = "I write user details to a .local file.";
+    this.permission = { topic: "system", mode: "write" };
+    this.inputs = {
+      email: { required: true },
+      company: { required: true },
+      subscribed: { required: false },
+    };
+    this.outputExample = {};
+  }
+
+  async runWithinTransaction({ params }) {
+    await ConfigUser.create(params);
```

Comment:
Should this action fail if `GROUPAROO_RUN_MODE !== 'cli:config'`?

Reply:
Yep! Added specs to ensure this is happening.